src/App.js

```diff
@@ -1,14 +1,15 @@
 import {
-  BrowserRouter as Router, Routes, Route, Link,
+  BrowserRouter as Router, Routes, Route,
 } from 'react-router-dom';
 import Books from './pages/Books';
 import Categories from './pages/Categories';
+import Navbar from './components/Navbar';
+import './App.css';
 
 function App() {
   return (
     <Router>
-      <Link to="/">Books</Link>
-      <Link to="/categories">Categories</Link>
+      <Navbar />
       <Routes>
         <Route path="/" exact element={<Books />} />
         <Route path="/categories" exact element={<Categories />} />
```

Comment:
- [ ] [REQUIRED] Please add style to your `Categories` section!

![image](https://user-images.githubusercontent.com/63915024/185466015-54f97681-8f23-4311-bb23-dbf6b27dc514.png)